ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/activity/TutorialActivity.kt

```diff
@@ -79,7 +79,7 @@ class TutorialActivity : BaseActivity<TutorialActivityBinding>(), TutorialCallba
     }
 
     override fun onOptionsItemSelected(item: MenuItem) = when (item.itemId) {
-        R.id.onboarding_action_skip -> {
+        R.id.onboarding_action_skip, R.id.action_tips_skip -> {
```

Comment:
I assume skip logic is already implemented, and this is actually just letting that logic actually run once the skip menu item is pressed for tutorial tips. So is it safe to assume if you didn't add this, and I tried pressing skip that it would do nothing?

Reply:
yeah, I opted to provide unique ids to the menu items for different tutorials that way if the logic ever needs to diverge I'm ready to handle the unique case for tips vs onboarding